colin-api/src/colin_api/models/business.py

```diff
@@ -123,6 +133,25 @@ def as_dict(self) -> Dict:
             }
         }
 
+    def as_slim_dict(self) -> Dict:
+        """Return slim dict version of self."""
+        return {
+            'business': {
+                'businessNumber': self.business_number,
+                'corpState': self.corp_state,
+                'corpStateClass': self.corp_state_class,
+                'foundingDate': self.founding_date,
+                'identifier': self.corp_num,
+                'jurisdiction': self.jurisdiction,
+                'homeRecognitionDate': self.home_recogn_dt,
+                'homeJurisdictionNumber': self.home_juris_num,
+                'homeCompanyName': self.home_company_nme,
+                'legalName': self.corp_name,
+                'legalType': self.corp_type,
+                'status': self.status
+            }
```

Comment:
What about `CAN_JUR_TYPE_CD` and `OTHR_JURIS_DESC`?

Reply:
it's handled here already I think - https://github.com/bcgov/lear/blob/cc383f277e190b4536a35aea80b3eb3c3e5d7905/colin-api/src/colin_api/models/business.py#L254-L262.  Unless you need that exposed directly

Reply:
OK, this looks OK already. Resolving this comment.